### PR TITLE
[stable/grafana] Make envFromSecret value be templated

### PR DESCRIPTION
**Pull request by atanasovskib**
The grafana chart provides a good way to supply
extra environment variables that can be loaded
from an already existing secret by specifying the
'envFromSecret' value to the name of an existing
secret object.

Making this be templated allows combining the grafana
chart in the same Release with other charts that would
generate that secret.

Signed-off-by: Blagoj Atanasovski <blagoj@timescale.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by blagojts** [comment]
@zanhsieh,  @rtluckie, or @maorfr is a change like this reasonable, can you take a look?

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.13
+version: 5.0.14
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: Could you bump to 5.0.16 please? @blagojts

Comment (atanasovskib) [reply stable/grafana/Chart.yaml]: @zanhsieh bumbed to 5.0.16

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**